### PR TITLE
test(e2e): suite: make recovery test work

### PR DESCRIPTION
**Pull request by mroz22**
this test has been red for a while https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/5389148910

### CODE REVIEW

File: packages/suite-web/e2e/tests/onboarding/analytics-consent.test.ts [hunk 1]
```diff
@@ -89,20 +73,6 @@ describe('Onboarding - analytics consent', () => {
         cy.getTestElement('@settings/menu/close').should('be.visible');
     });
 
-    it('shows analytics consent when going to settings and back on initialized T2B1 device', () => {
```
Comment (mroz22) [inline packages/suite-web/e2e/tests/onboarding/analytics-consent.test.ts:92]: this analytics-consent test is written in unit-test way. this is not right, and if we wanted to test every scenario with every device we would soon explode in complexity. removing model R variants

File: packages/suite-web/e2e/tests/onboarding/analytics-consent.test.ts [hunk 2]
```diff
@@ -20,12 +20,6 @@ const acceptAnalyticsConsentOnInitializedDevice = () => {
 };
 
 describe('Onboarding - analytics consent', () => {
-    const startEmuOpts = {
```
Comment (mroz22) [inline packages/suite-web/e2e/tests/onboarding/analytics-consent.test.ts:23]: this is old anyway, model R emus should now be run from master

File: packages/suite-web/e2e/tests/onboarding/t2t1-create-wallet.test.ts [hunk 3]
```diff
@@ -8,78 +8,11 @@ describe('Onboarding - create wallet', () => {
         cy.prefixedVisit('/');
     });
 
-    // cypress open todo
-    it('Success (no shamir capability)', () => {
-        cy.task('startEmu', { wipe: true, version: '2.1.1' });
```
Comment (mroz22) [inline packages/suite-web/e2e/tests/onboarding/t2t1-create-wallet.test.ts:13]: this test does not work. maybe trezorlib update. I dont know. but we were testing 2.1.1 which is now definitely more than 2 years old

File: packages/suite-web/e2e/tests/onboarding/t2t1-create-wallet.test.ts [hunk 4]
```diff
@@ -8,78 +8,11 @@ describe('Onboarding - create wallet', () => {
...
-        );
-        cy.getTestElement('@onboarding/confirm-on-device');
-        cy.getTestElement('@components/collapsible-box').should('have.css', 'opacity', '1');
-        cy.task('pressYes');
-        cy.task('inputEmu', '1');
-        cy.task('inputEmu', '1');
-    });
-
-    // cypress open todo: re-enable
-    it.skip('Success (Shamir backup)', () => {
```
Comment (mroz22) [inline packages/suite-web/e2e/tests/onboarding/t2t1-create-wallet.test.ts:78]: and this was skipped, so when talking about tests count, no difference here

File: packages/suite-web/e2e/tests/onboarding/t2t1-recovery-fail.test.ts [hunk 5]
```diff
@@ -6,12 +6,12 @@ describe('Onboarding - recover wallet T2T1', () => {
         cy.task('startBridge');
         cy.viewport(1080, 1440).resetDb();
         cy.prefixedVisit('/');
-        cy.task('startEmu', { wipe: true, version: '2.4.3' });
```
Comment (mroz22) [inline packages/suite-web/e2e/tests/onboarding/t2t1-recovery-fail.test.ts:9]: if there is no good reason to test specific version, we should always  test the latest one

File: packages/suite-web/e2e/tests/onboarding/t2t1-recovery-success.test.ts [hunk 6]
```diff
@@ -7,7 +7,7 @@ describe('Onboarding - recover wallet T2T1', () => {
         cy.viewport(1080, 1440).resetDb();
         cy.prefixedVisit('/');
 
-        cy.task('startEmu', { wipe: true, version: '2.4.3' });
```
Comment (mroz22) [inline packages/suite-web/e2e/tests/onboarding/t2t1-recovery-success.test.ts:10]: here we should also test the latest one but I believe I was hit by this bug https://satoshilabs.slack.com/archives/CL1D61PQF/p1698389109047929
cc @trezor/qa

**Review by matejkriz — APPROVED** [review]
👏

**Pull request merged**